src/kOS/Communication/ConnectivityManager.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace kOS.Communication
+{
+    public interface ConnectivityManager
+    {
+        double GetDelay(Vessel vessel1, Vessel vessel2);
+    }
+}
```

Comment:
I'm only wondering because of my work to get a command line instance running.  I'm of the mindset to put everything we can into Safe, that way we can run unit tests, check performance, and do faster debugging.  It isn't super critical right now, but I probably will want to get there eventually.